.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Index and build site
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+  schedule:
+  - cron:  "0 */2 * * *"
+
+jobs:
+  index:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js 12
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: Setup git user
+      run: |
+        git config user.email "<>"
+        git config user.name "${GITHUB_ACTOR}"
+    - name: Setup gh-pages branch
+      run: |
```

Comment:
This whole part should be replaced with https://github.com/pkgjs/gh-pages/

That is the whole reason I made that action in the first place.